CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [Unreleased]
+### Fixed
+- Velocity is now correctly displayed on project index
```

Comment:
We found another issue related to this item, I think this text is precipitated

Reply:
@Drowze Change this to something like: "Velocity isn't being retrieved only by fallback anymore"

Reply:
Updated the text! Please take a look now